CHANGELOG.md

```diff
@@ -32,6 +36,13 @@
     and provide other improvements.
   - Use global `URL` interface to handle relative redirects.
 
+### Removed
+- Experimental non-standard `protectedMode` option.
+- **BREAKING**: Various console warnings were removed. The newly added "safe
+  mode" can stop processing where these warnings were.
+- **BREAKING**: Remove `compactionMap` and `expansionMap`. Their known use
+  cases are addressed with "safe mode" and future planned features.
```

Comment:
Is this section in the right place? If so, ignore. If not... move :)

Reply:
Rebase bug.  Fixed.